tests/untl_structure_test.py

```diff
@@ -408,12 +401,12 @@ def test_Metadata_create_xml_string():
     description = us.Description(content=u'Adaption of "Fortuna te dé Dios, hijo"',
                                  qualifier=u'content')
```

Comment:
Let's get rid of all these old-style unicode string literals in this repo. I would recommend you make the changes together in a single commit that has no other types of changes so it's easier to review.

Reply:
sounds good! will do!